managers/muse.py

```diff
@@ -151,7 +151,7 @@ def constructS3Link(self, bucketLocation):
             self.s3Bucket, bucketLocation
         )
 
-    def createManifestInS3(self, bucketLocation):
+    def create_manifest_in_s3(self, bucketLocation):
         s3URL = self.constructS3Link(bucketLocation)
```

Comment:
I think we may want to start using `get_stored_file_url` in the digital_assets/utils folder. 

This'll make it work for your local env, too.

Reply:
Ok I'll add that method to other ingest processes.